### DISCUSSION TITLE
Need explanation about i3, GNOME and how setting works

### DISCUSSION THREAD

**Discussion by ngamsomset** [i3/i3#5920] [Q&A]
Hi there.

I'm new to i3 and I need a clarification on how setting works in i3 workspace and a default GNOME.

1. Correct me if I'm wrong, what every setting that I did in the default GNOME profile will NOT reflect in i3. (ig. setting screen resolution)

2. I use HP laptop with Fedora, I want to hook it to monitor with HDMI and start working right away with my laptop lid close. Where do I set the setting for that. At the moment, if I connect to external monitor, my 1st workspace always on my laptop screen and the external monitor will show 2 to start with.

**Comment by akshatdevp** [comment]
You can configure displays using xrandr(install via package manager). arandr as well if you want a GUI.
Deactivate your laptop screen
Make monitor primary
save the config
execute the config from your i3 config file so it stays like that after restart.(`exec --no-startup-id /.screenlayout/default.sh`)
